modules/user_accounts/php/useraccountrowprovisioner.class.inc

```diff
@@ -17,6 +17,18 @@ class UserAccountRowProvisioner extends \LORIS\Data\Provisioners\DBRowProvisione
      */
     function __construct()
     {
+        // Get user
+        $factory = \NDB_Factory::singleton();
+        $user    = $factory->user();
+        if (!$user->hasPermission('user_accounts_multisite')) {
+            $siteLimitQuery = "LEFT JOIN user_psc_rel upsr2
+                ON (upsr2.CenterID=upsr.CenterID)
+                WHERE upsr2.UserID=:uid";
+            $params         = ['uid' => $user->getId()];
+        } else {
+            $siteLimitQuery = '';
+            $params         = [];
+        }
```

Comment:
I don't understand why this is necessary.

user_accounts.class.inc already has:
```
    /**
     * Tells the base class that this page's provisioner can support the
     * HasAnyPermissionOrUserSiteMatch filter.
     *
     * @return ?array of permissions or null
     */
    public function allSitePermissionNames() : ?array
    {
        return ['user_accounts_multisite'];
    }
```

so shouldn't the HasAnyPermissionOrUserSiteMatch filter take care of it?